tests/utils/command.rs

```diff
@@ -0,0 +1,33 @@
+use assert_cmd::cargo::CommandCargoExt;
+use std::process::Command;
+
+pub fn bat_raw_command_with_config() -> Command {
+    let mut cmd = Command::cargo_bin("bat").unwrap();
+    cmd.current_dir("tests/examples");
+    cmd.env_remove("BAT_CACHE_PATH");
+    cmd.env_remove("BAT_CONFIG_DIR");
+    cmd.env_remove("BAT_CONFIG_PATH");
+    cmd.env_remove("BAT_OPTS");
+    cmd.env_remove("BAT_PAGER");
+    cmd.env_remove("BAT_STYLE");
+    cmd.env_remove("BAT_TABS");
+    cmd.env_remove("BAT_THEME");
+    cmd.env_remove("COLORTERM");
+    cmd.env_remove("NO_COLOR");
+    cmd.env_remove("PAGER");
+    cmd
+}
+
+pub fn bat_raw_command() -> Command {
```

Comment:
I think this function (or the whole module) should be `#[cfg(test)]`, to silence the warnings. Sorry, just saw this now.

Similar for the other functions in this file.

Reply:
I've tried putting `#[cfg(test)]` on there, but `cargo test` still emits the warning when building.
Or did you mean some other warnings?

Reply:
After looking at it again, I guess rust still emits a warning, as the function is only
used in unix tests. I'm not 100% sure how to fix this though, as one cannot put multiple
`cfg` annotations on top. Do you maybe have an idea?

Reply:
#[cfg(test)] is not enough (but does not hurt), because the mod is brought into scope by a test, but then never used. There are ways to restructure the code, but I don't think it is worth it, so let's just allow it, at least for now.